examples/bri-3/src/bri/workgroup/workgroups/capabilities/updateWorkgroup/updateWorkgroupCommand.handler.ts

```diff
@@ -9,17 +10,18 @@ export class UpdateWorkgroupCommandHandler
 {
   constructor(
     private agent: WorkgroupAgent,
-    private storageAgent: WorkgroupStorageAgent,
+    private workgroupStorageAgent: WorkgroupStorageAgent,
+    private bpiSubjectStorageAgent: BpiSubjectStorageAgent,
   ) {}
 
   async execute(command: UpdateWorkgroupCommand) {
     const adminstratorsToUpdate =
-      await this.agent.fetchWorkgroupAdministratorsAndThrowIfNoneExist(
+      await this.bpiSubjectStorageAgent.getBpiSubjectsById(
         command.administratorIds,
```

Comment:
Since both adminstratorIds and participantIds are user inputs, i would do a check here to make sure they are all present before updating the workgroup and would otherwise throw. For example, If we swallow the wrong Id the user might be unaware that the new admin has not been added to the group which could lead to a problem if the decides to ditch the old admin identity for whatever reason.

Reply:
Implemented as length check